python/jupytergis_server/jupytergis_server/server_extension.py

```diff
@@ -0,0 +1,21 @@
+from .handler import JupyterGISHandler
+from jupyter_server.utils import url_path_join
+
+
+def _load_jupyter_server_extension(serverapp):
+    """
+    This function is called when the extension is loaded.
+    """
+
+    web_app = serverapp.web_app
+    host_pattern = ".*$"
+    route_pattern = url_path_join(web_app.settings["base_url"], "/jupytergis/(.*)")
```

Comment:
We should probably make sure it does not collide with the route we already define in the current server extension (let's merge both server extensions into this new package `jupytergis_server`?)

Reply:
I had actually not seen you had a server extension in `jupytergis_core`, what is it used for?

Reply:
It's a simple server proxy to bypass CORS issues on some online assets

Reply:
I'd be curious to get @mfisher87 on this